xml/System/TimeSpan.xml

```diff
@@ -851,9 +851,9 @@
         <Parameter Name="ts" Type="System.TimeSpan" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="ts">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ts">The value to be divided by.</param>
+        <summary>Returns a new <see cref="T:System.Double" /> value which is the result of division of this instance and the specified <paramref name="ts" />.</summary>
+        <returns>A new value that represents result of division of this instance by the value of the <paramref name="ts" />.</returns>
```

Comment:
>value [](start = 23, length = 5)

should be ticks value

Reply:
isn't the result unit-less? We divide time by time so you should get a ratio back

Reply:
it is not ration more than real ticks. If you are dividing 2 times what the result means to you? and how you can use it later? it should have some meaning. right?

Reply:
I see, I think this comment was meant to be on the overload above (TimeSpan/double). For this one (TimeSpan/TimeSpan) 4s divided by 2s means it's 2 times longer

Reply:
(note the above one mentions TimeSpan object not `value`)